lib/opFns.js

```diff
@@ -908,8 +921,12 @@ function makeCall (runState, callOptions, localOpts, cb) {
       for (var i = 0; i < Math.min(localOpts.outLength, results.vm.return.length); i++) {
         runState.memory[localOpts.outOffset + i] = results.vm.return[i]
       }
+
+      if (['CALL', 'CALLCODE', 'DELEGATECALL'].indexOf(runState.opName) >= -1) {
```

Comment:
L925 should be rewritten as:
`      switch(runState.opName) {
        case 'CALL':
        case 'CALLCODE':
        case 'DELEGATECALL':
          runState.lastReturned = results.vm.return
      }   `

Easier to understand.  Probably faster as well.